atmosphere.newmann_zarr dataset fails to load

Our CI has identified a problem with the following dataset

https://github.com/pangeo-data/pangeo-datastore/blob/eea12c82a4f1c5afe34fb5e9696813900174b07a/intake-catalogs/atmosphere.yaml#L6-L14

```
_____________ test_intake_dataset_to_dask[atmosphere.newmann_zarr] _____________
catalog = <Intake catalog: master>
dataset_name = ('atmosphere', 'newmann_zarr')
    def test_intake_dataset_to_dask(catalog, dataset_name):
        item = get_nested_item(catalog, dataset_name)
>       ds = item.to_dask()
tests/validate_datasets.py:45: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/srv/conda/lib/python3.6/site-packages/intake_xarray/base.py:70: in to_dask
    return self.read_chunked()
/srv/conda/lib/python3.6/site-packages/intake_xarray/base.py:45: in read_chunked
    self._load_metadata()
/srv/conda/lib/python3.6/site-packages/intake/source/base.py:117: in _load_metadata
    self._schema = self._get_schema()
/srv/conda/lib/python3.6/site-packages/intake_xarray/base.py:19: in _get_schema
    self._open_dataset()
/srv/conda/lib/python3.6/site-packages/intake_xarray/xzarr.py:35: in _open_dataset
    self._ds = xr.open_zarr(self._mapper, **self.kwargs)
/srv/conda/lib/python3.6/site-packages/xarray/backends/zarr.py:447: in open_zarr
    group=group, consolidated=consolidated)
/srv/conda/lib/python3.6/site-packages/xarray/backends/zarr.py:250: in open_group
    zarr_group = zarr.open_group(store, **open_kwargs)
/srv/conda/lib/python3.6/site-packages/zarr/hierarchy.py:1117: in open_group
    err_group_not_found(path)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
path = ''
    def err_group_not_found(path):
>       raise ValueError('group not found at path %r' % path)
E       ValueError: group not found at path ''
```

In the storage browser, it looks like this dataset isn't there any more.

@jhamman - Any idea what's up? I think you uploaded this one, no?

--- a/intake-catalogs/atmosphere.yaml
+++ b/intake-catalogs/atmosphere.yaml
@@ -3,16 +3,6 @@ plugins:
     - module: intake_xarray
 sources:
 
-  newmann_zarr:
-    description: Newman met ensemble in zarr format (9 ens members)
-    driver: zarr
-    args:
-      storage_options:
-        project: 'pangeo-181919'
-        token: anon
-        access: read_only
-      urlpath: 'gcs://pangeo-data/newman-met-ensemble'
-
   gmet_v1:
     description: Full GMET version 1 (Newman) met ensemble in zarr format
     driver: zarr

